default/scripting/species/common/population.macros

```diff
@@ -65,7 +66,8 @@ GOOD_POPULATION
             scope = Source
             activation = Planet
             priority = [[TARGET_POPULATION_SCALING_PRIORITY]]
-            effects = SetTargetPopulation value = Value +0.25*abs(Value) accountinglabel = "GOOD_POPULATION_LABEL"
+            effects = SetTargetPopulation value = Value +0.25*abs(Value)
+            accountinglabel = "GOOD_POPULATION_LABEL"
```

Comment:
etc

Reply:
ok i can reverse those chunks. just seemed to be clearer readable in code. how about the cases where there was no accounting label before, should it still go on the same line as the effect? what if the effect goes over multiple lines?

Reply:
it's much less clear, as it presents the accounting label as not part of the effect.

if it can just go into the effectsgroup, rather than into one of the `SetMeter` effects, then put it before the `effects =` line

Reply:
well i did that in garrison and computronium, is where unittest number 5 failed with parse errors when i put it between priority and effects on its own line

Reply:
i'll fix it tomorrow, and there will be more accounting labels to adjust, for sure.